saleor/checkout/complete_checkout.py

```diff
@@ -307,11 +307,11 @@ def _create_line_for_order(
     )
 
     line_voucher_code = None
-    if checkout_line_info.voucher:
-        line_voucher_code = checkout_line_info.voucher.code
+    if voucher := checkout_line_info.voucher:
+        line_voucher_code = voucher.last_code
     order_voucher_code = None
-    if checkout_info.voucher:
-        order_voucher_code = checkout_info.voucher.code
+    if voucher := checkout_info.voucher:
+        order_voucher_code = voucher.last_code
```

Comment:
We should get the `voucher_code` from `checkout` here. If `checkout_line` has a `voucher` instance it means that the code is `lint_voucher_code`, otherwise is a `order_voucher_code`

Reply:
Not resolved